README.md

```
@@ -1 +1,166 @@
+
+- **Email**: vatsal.sanjay@comphy-lab.org
+- **Website**: [comphy-lab.org](https://comphy-lab.org)
+- **ORCID**: [0000-0002-4293-6099](https://orcid.org/0000-0002-4293-6099)
+
+## License
+
+This CV template uses the moderncv LaTeX package. The content is personal academic information of Vatsal Sanjay.
+
+## Legacy Note
+
+For the customized CV option used in PhD applications, please visit https://github.com/VatsalSy/Vatsal_CV/tree/PhDapplication
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)